daterangepicker.js

```diff
@@ -23,14 +23,13 @@
       if (!jQuery) throw new Error('jQuery dependency not found');
     }
 
-    factory(root, exports, momentjs, jQuery);
-
+    module.exports = factory(root, momentjs, jQuery);
   // Finally, as a browser global.
   } else {
-    root.daterangepicker = factory(root, {}, root.moment || moment, (root.jQuery || root.Zepto || root.ender || root.$));
+    root.daterangepicker = factory(root, root.moment, (root.jQuery || root.Zepto || root.ender || root.$));
```

Comment:
Here you change the function signature and remove `moment`, which is needed by Meteor.js users as the `moment` object is not attached to the window (root) for them. This change would mean they have to go back to maintaining a fork instead of using this package.